script/Ciel/Day 12/12_01_CIEL12_3_2B.txt

```diff
@@ -76,50 +86,59 @@ Right, I'll permit it just for today. You should be taking proper care of Shiki-
 [sha:6eaa101c90b0a9a4a056fe04652b475a15930f72]{
 -- Page 199, Offset 31414.
 -- 「そうですね……診たところ、志貴さんの熱は疲れからきているようですから、薬はわたしが処方します。
-"Hmm... back when I examined him, it seemed like his fever was caused by fatigue, so I'll go prepare the medicine.
+"Let me think...
+ on examination it seemed like his fever was the result of fatigue,
+ so I'll prepare an appropriate remedy.
 }
 [sha:0a364cc011541171e6f87725760b8d1087181275]{
 -- Page 199, Offset 31415.
 -- 翡翠ちゃん、すぐに行くからあとはよろしくね」
-Hisui-chan, I'm going there right now, so I'll leave the rest to you."
+Hisui-chan, I'm going to go see to that right away. I leave the rest to you."
 }
 [sha:395720a3868f2771d28d7c2e148b7543b5101ff8]{
 -- Page 200, Offset 31416.
 -- 琥珀さんは早足で屋敷の西館……彼女の部屋がある棟へと歩いていく。
-Kohaku-san briskly walks to the west wing, where her room is located.
+Kohaku-san briskly heads off toward the west wing, where her room is located.
 }
 [sha:26622c43242bf2b45201689aa6e60b6d158c6811]{
 -- Page 201, Offset 31417.
 -- 「それでは志貴さま、お部屋にお連れします」
-"In that case, Shiki-sama. I'll accompany you to your room."
+"Very well then. Shiki-sama... I will accompany you to your room."
 }
 [sha:e29b66015700620face678866ffe44713c478406]{
 -- Page 201, Offset 31418. Mods: @n.
 -- 失礼します、と断ってから、
-After expressing her courtesies―――
+After apologizing in advance,
 }
 [sha:163daad24228d15ecb28d81a31a56ba00321295d]{
 -- Page 201, Offset 31419. Glued.
 -- 翡翠はおそるおそる俺に肩を貸してきた。
-Hisui timidly lends me her shoulder.
+ Hisui timidly lends me her shoulder.
 }
 [sha:5539a2e0f92f0e7e0b7db600b742b8b25c3aa0a3]{
 -- Page 202, Offset 31420.
 -- ……翡翠の手をかりて階段を上がっていく。
-...I take her hand and begin climbing up the stairs.
+With her help, I begin climbing the stairs.
 }
 [sha:df485d686e094a1d6add6f92de9a6ed57733f605]{
 -- Page 202, Offset 31421.
 -- 途中。ギシギシと繰り返される頭痛より、すぐ真横にいる翡翠の辛そうな顔の方が、ひどく記憶に焼きついた。
-As we make our way up, it isn't my creaking headache, but rather Hisui's pained expression as she stands beside me that's seared into my memories.
-}
-[sha:8b01a2d4245fa58bdd660d07fd69d37ff2b04fc7]{
+// Some tinkering done with 記憶 here - this is a past aspect sentence,
+// so he's thinking back on how the thing he remembers about climbing the stairs isn't the pain but Hisui's face,
+// but in general we prefer active sentences, so we're still mid-stair climb in EN.
+// As such, rendered as thoughts rather than memories.
+As we make our way upstairs,
+ it isn't the constant pounding of my headache that dominates my thoughts.
+ It's the pained expression on Hisui's face as she stands right beside me.
```

Comment:
maybe instead of 'dominates my thoughts' something like 'leave an impression' could be appropriate? Since that's more associated with memory stuff

Oh and semi-colon is very appropriate here, imo.

>...of my headache that leaves an impression; it's the...